Assets/HoloToolkit/SpatialSound/Scripts/UAudioManager/SpatialSoundSettings.cs

```diff
@@ -61,29 +59,29 @@ public static void SetRoomSize(AudioSource audioSource, SpatialSoundRoomSizes ro
         /// </summary>
         /// <param name="audioSource">The AudioSource on which the minimum gain will be set.</param>
         /// <param name="room">The desired minimum gain, in decibels.</param>
+        [Obsolete("This parameter is no longer used, use the unity volume curve.")]
```

Comment:
Awesome!  Could you also call out the `ClassName.MethodName()` in this way please?

Reply:
Updated the warning text.

Reply:
There isn't really a new function, volume attenuation is now controlled by the unity volume curve on the audio source.  This is now a parameter in the audio event and replaces the Min/Max gain and UnityGainDistance parameters that used to be in the audio event.